docs/tutorials/ui-customization.md

```diff
@@ -163,6 +163,17 @@ const config = {
 ui.configure(config);
 ```
 
+#### Configuring playback, fast forward and rewind rates
+The rate in which the player can play, fast forward and rewind content can be configured using the `playbackRates`, `fastForwardRates` and `rewindRates` options.
```

Comment:
Can you add more explanation for what the difference between these are?  It's not clear why you have both `playbackRates` and separate fast-forward and rewind rates.